packages/blockchain-wallet-v4-frontend/src/modals/ShowUsedAddresses/index.js

```diff
@@ -4,28 +4,34 @@ import { bindActionCreators, compose } from 'redux'
 
 import { actions } from 'data'
 import modalEnhancer from 'providers/ModalEnhancer'
-import ShowUsedAddresses from './template.js'
+import ShowUsedAddresses from './template'
 
 class ShowUsedAddressesContainer extends React.PureComponent {
   constructor (props) {
     super(props)
+    this.state = { busy: false }
```

Comment:
?

Reply:
used to indicate loading to the user during the address derivation process